packages/react/src/components/Table/TableBody/TableBodyLoadMoreRow/_table-body-load-more-row.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../../globals/vars';
+
+.#{$iot-prefix}--load-more-cell {
```

Comment:
I would rather see us use a more qualified selector rather than the `!important` here. Something like `.#{$iot-prefix}--table-container .#{$iot-prefix}--load-more-cell`. That way we don't have issues with specificity in the future.